includes/class-donations.php

```diff
@@ -428,11 +423,6 @@ public static function get_donation_settings() {
 			$parsed_settings['amounts'][ $frequency ] = array_map( 'floatval', $amounts );
 		}
 
-		// Ensure a minimum donation amount is set.
-		if ( ! isset( $saved_settings['minimumDonation'] ) && self::is_platform_wc() ) {
-			self::update_donation_product( [ 'minimumDonation' => $settings['minimumDonation'] ] );
-		}
-
```

Comment:
This was added in #1895, as a migration step. By now we can be sure everything was migrated.